packages/color-wheel/src/spectrum-config.v2.js

```diff
@@ -24,6 +27,10 @@ const config = {
             inPackage: '@spectrum-css/colorwheel',
             outPackage: 'color-wheel',
             fileName: 'color-wheel',
+            excludeByComponents: [
+                builder.class('🤫'),
```

Comment:
?

Reply:
CSS needs this to not make their tooling angry and then it ends up in our CSS if we don't exclude it 😞.